docs/updating/updating_to_3.3.md

```diff
@@ -279,6 +279,18 @@ You do this manually by following this procedure:
 
     `php bin/console doctrine:schema:update --em=<ENTITY_MANAGER_NAME> --force`
 
+#### VCL configuration for Fastly
+
+If you use Fastly, update your VCL configuration.
+
+Locate the `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_main.vcl` file and add the following lines to it:
```

Comment:
Do we need to say where exactly in the file one must add these lines? Or it is clear to the reader?